### PR TITLE
Fix local-run healthchecks when using custom soadir argument

### PR DESCRIPTION
**Pull request by keshavdv**

### CODE REVIEW

File: tests/cli/test_cmds_local_run.py [hunk 1]
```diff
@@ -344,7 +349,8 @@ def test_configure_and_run_pulls_image_when_asked(
         command=['bash'],
         healthcheck=args.healthcheck,
         healthcheck_only=args.healthcheck_only,
-        instance_config=mock_get_instance_config.return_value
+        instance_config=mock_get_instance_config.return_value,
+        soa_dir=args.yelpsoa_config_root
```
Comment (EvanKrall) [inline tests/cli/test_cmds_local_run.py]: go ahead and add a trailing comma so we don't get a 3-line diff the next time someone adds an arg.

File: tests/cli/test_cmds_local_run.py [hunk 2]
```diff
@@ -673,6 +679,57 @@ def test_run_docker_container_non_interactive_run_returns_nonzero(
...
+            'fake_service',
+            'fake_instance',
+            'fake_hash',
+            [],
+            False,  # interactive
+            'fake_command',
+            False,  # healthcheck
+            True,  # terminate after healthcheck
+            mock_service_manifest,
+            soa_dir='fake_soa_dir'
```
Comment (EvanKrall) [inline tests/cli/test_cmds_local_run.py]: + trailing comma

File: tests/cli/test_cmds_local_run.py [hunk 3]
```diff
@@ -673,6 +679,57 @@ def test_run_docker_container_non_interactive_run_returns_nonzero(
...
+        )
+    assert mock_docker_client.stop.call_count == 1
+    assert mock_docker_client.remove_container.call_count == 1
+    assert excinfo.value.code == 0
+    mock_get_healthcheck_for_instance.assert_called_with(
+        'fake_service',
+        'fake_instance',
+        mock_service_manifest,
+        mock_pick_random_port.return_value,
+        soa_dir='fake_soa_dir'
```
Comment (EvanKrall) [inline tests/cli/test_cmds_local_run.py]: trailing comma

File: tests/cli/test_cmds_local_run.py [hunk 4]
```diff
@@ -296,7 +299,8 @@ def test_configure_and_run_uses_bash_by_default_when_interactive(
         command=['bash'],
         healthcheck=args.healthcheck,
         healthcheck_only=args.healthcheck_only,
-        instance_config=mock_get_instance_config.return_value
+        instance_config=mock_get_instance_config.return_value,
+        soa_dir=args.yelpsoa_config_root
```
Comment (EvanKrall) [inline tests/cli/test_cmds_local_run.py]: trailing comma

File: tests/cli/test_cmds_local_run.py [hunk 5]
```diff
@@ -254,7 +255,8 @@ def test_configure_and_run_command_uses_cmd_from_config(
         command=shlex.split(mock_get_instance_config.return_value.get_cmd.return_value),
         healthcheck=args.healthcheck,
         healthcheck_only=args.healthcheck_only,
-        instance_config=mock_get_instance_config.return_value
+        instance_config=mock_get_instance_config.return_value,
+        soa_dir=args.yelpsoa_config_root
```
Comment (EvanKrall) [inline tests/cli/test_cmds_local_run.py]: comma

File: paasta_tools/cli/cmds/local_run.py [hunk 6]
```diff
@@ -400,7 +400,8 @@ def run_docker_container(
     command,
     healthcheck,
     healthcheck_only,
-    instance_config
+    instance_config,
+    soa_dir=service_configuration_lib.DEFAULT_SOA_DIR
```
Comment (EvanKrall) [inline paasta_tools/cli/cmds/local_run.py]: comma

**Comment by EvanKrall** [comment]
Other than minor style things, looks good. Welcome back!

**Comment by solarkennedy** [comment]
Cool. @keshavdv, fix the style things and then on github after a shipit you should merge it yourself.

File: tests/cli/test_cmds_local_run.py [hunk 7]
```diff
@@ -673,6 +679,57 @@ def test_run_docker_container_non_interactive_run_returns_nonzero(
...
             return_value=('fake_healthcheck_mode', 'fake_healthcheck_uri'),
             )
+def test_run_docker_container_with_custom_soadir_uses_healthcheck(
+    mock_get_healthcheck_for_instance,
+    mock_get_container_id,
+    mock_run,
+    mock_execlp,
+    mock_get_docker_run_cmd,
+    mock_pick_random_port,
+    mock_simulate_healthcheck
```
Comment (nhandler) [inline tests/cli/test_cmds_local_run.py]: trailing comma

**Comment by nhandler** [comment]
I'm going to go ahead and merge this pull request. I'll fix that one last issue with a separate commit.

**Pull request merged**